src/inference/src/dev/core_impl.cpp

```diff
@@ -1478,16 +1478,16 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::load_model_from_cache(
                 ov::AnyMap update_config = config;
                 update_config[ov::loaded_from_cache.name()] = true;
 
+                if (util::contains(plugin.get_property(ov::supported_properties), ov::hint::model) &&
```

Comment:
Should be some test be added that checks this property is not passed to plugin?

Reply:
The test will be added in https://github.com/openvinotoolkit/openvino/pull/29354